runtimes/node-18.0/src/server.js

```diff
@@ -5,20 +5,40 @@ const { text: parseText, json: parseJson, send } = require("micro");
 
 const USER_CODE_PATH = '/usr/local/server/src/function';
 
+const randomId = () => {
+    return `${new Date().getTime().toString(16)}${Math.round(Math.random() * 1000000000).toString(16)}`;
+}
+
+const writeLogs = (id, type, log) => {
+    fs.appendFileSync(`/mnt/logs/${id}_${type}.log`, log);
```

Comment:
instead of writing to a file like this why not use a more native approach 

```
var logsStdout = fs.createWriteStream('mnt/logs/${id}_stdout.log');
var logsStderr = fs.createWriteStream('mnt/logs/${id}_stderr.log');
process.stdout.write = logsStdout.write.bind(logsStdout);
process.stderr.write = logsStderr.write.bind(logsStderr);
```

Reply:
Discussed together, we will be keeping approach, but will use streams instead

Reply:
Applied stream approach